index.html

```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+      <div class="card card-with-bgimage">
+        <h2>Profesional Art Printing Data</h2>
+        <p>
+          A daily selection of privately personalized reads; no accounts or sign-ups required. 
+          has been the industry's standard
+        </p>
+        <ul>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+        </ul>
+        <button type="button">
+          See Project
+        </button>
+      </div>
+
+    </div>
+  </section>
+
+  <section class="about-me">
+    <h1>About me</h1>
+    <p>
+      Hello I’m a software developer! I can help you build a product , 
+      feature or website Look through some of my work and experience! 
+      If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.
+    </p>
+    <button type="button">Get My Resume</button>
+    <hr>
+
+    <div class="title-container"><div class="diamond-icon"></div><h2>Languages</h2></div>
+    <ul>
+      <li>Javascript</li>
+      <li>Ruby</li>
+      <li>HTML</li>
+      <li>CSS</li>
+    </ul>
+
+    <div class="title-container"><div class="square-icon"></div><h2>Frameworks</h2></div>
+    <ul>
+      <li>Bootstrap</li>
+      <li>Ruby on Rails</li>
+      <li>RSPec</li>
+      <li>CapyBara</li>
```

Comment:
- [ ] Please, consider that the "About me" section has to be fixed by alignment - please, refer to the Figma design
> Your version
<img width="234" alt="Screen Shot 2022-03-26 at 20 46 31" src="https://user-images.githubusercontent.com/49064106/160249397-9254bb31-b92b-4bf1-923b-4aa4b38a42ab.png">

> Figma version
<img width="157" alt="Screen Shot 2022-03-26 at 20 43 03" src="https://user-images.githubusercontent.com/49064106/160249403-9e8f2636-1793-4d39-b399-8ddfc05fbced.png">

- [ ] Please, consider the "About me" section paragraph content is a little bit hard to read - try to fix the position of the paragraph, to not cover the background
<img width="239" alt="Screen Shot 2022-03-26 at 20 50 57" src="https://user-images.githubusercontent.com/49064106/160249487-add92d7a-b652-43b2-9235-364c097604aa.png">